wp-parsely.php

```diff
@@ -103,6 +104,20 @@ function parsely_admin_menu_register(): void {
 	$settings_page->run();
 }
 
+require __DIR__ . '/src/class-recommended-content.php';
+
+require __DIR__ . '/src/Endpoints/class-recommendations-api-proxy.php';
+/**
+ * Register the Recommendatiosn API Proxy WP-API REST Endpoint
+ *
+ * @return void
+ */
+function rest_api_init_recommendations_proxy() {
+	$api = new Recommendations_API_Proxy();
+	$api->run();
+}
+add_action( 'rest_api_init', __NAMESPACE__ . '\\rest_api_init_recommendations_proxy' );
```

Comment:
Should this be feature flagged?

Reply:
Yes, I would put a filter inside the `run` method.